sonar.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/sh
+
+echo "Building docker image"
+docker build -f ./config/sonar.dockerfile -t sonar ./config/
+echo "Running docker container"
+docker run -dit -p 9000:9000 --name sonar sonar:latest
```

Comment:
@romani I'm not really familiar with wercker ... I only took a quick look into it.
Are we going to create a separate wercker.yml which will be based on sonarqube box? (Cause if so, it will have no need to stop container also...)